homeassistant/components/cover/rflink.py

```diff
@@ -0,0 +1,119 @@
+"""
+Support for Rflink Cover devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/cover.rflink/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.rflink import (
+    DATA_ENTITY_GROUP_LOOKUP, DATA_ENTITY_LOOKUP,
+    DEVICE_DEFAULTS_SCHEMA, EVENT_KEY_COMMAND, RflinkCommand)
+from homeassistant.components.cover import (
+    CoverDevice, PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_NAME
+DEPENDENCIES = ['rflink']
```

Comment:
I'm not sure if it is really something that's enforced/wanted on homeassistant in general, but I would separate variable declarations from imports for clarity, having (at least) one newline between them.